config/zeus-bolt.php

```diff
@@ -63,6 +63,10 @@
 
     'allow_design' => false,
 
+    'editable_forms' => false,
+
+    'form_action_position_tailwind_class' => 'text-center',
```

Comment:
kind don't like this, its means have to add a config file to the tailwind css `content` array. currently I recommend to publish the views and edit any needed classes. especially for the FE views.

Reply:
Thanks for the feedback - personally, I think having to maintain all the views for the sake of a button position is a bit excessive. Can you think of a middle ground?

Reply:
I get your point
how about make it `null`. by default, and add `text-center` to the blade:
`config('....','text-center')`
and add a comment to remind the users to add the config file to tailwind css.

Reply:
this will also assure there is no breaking change if the user didnt update the published config file